### PR TITLE
Add missed lines in 12-bit formats support.

### PR DESCRIPTION
**Pull request by onabiull**
Fixes: 444ecb48 ("Add 12-bit format support")

### CODE REVIEW

File: _studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp [hunk 1]
```diff
@@ -1650,6 +1678,16 @@ mfxStatus CheckVideoParam(MfxVideoParam& par, MFX_ENCODE_CAPS_HEVC const & caps,
 
     switch (CO3.TargetBitDepthLuma)
     {
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+    case 12:
+        invalid += CheckOption(CO3.TargetBitDepthChroma, 12/*, 10, 8*/, 0);
+        invalid += CheckOption(par.mfx.CodecProfile
+//            , (mfxU16)MFX_PROFILE_HEVC_MAIN10
```
Comment (dvrogozh) [inline _studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp]: what this commented line is about?

File: _studio/shared/src/libmfx_core.cpp [hunk 2]
```diff
@@ -1381,7 +1381,45 @@ mfxStatus CoreDoSWFastCopy(mfxFrameSurface1 *pDst, mfxFrameSurface1 *pSrc, int c
 
         break;
 
+#if (MFX_VERSION >= 1027)
```
Comment (dvrogozh) [inline _studio/shared/src/libmfx_core.cpp]: this one is not terminated

File: _studio/shared/src/libmfx_core.cpp [hunk 3]
```diff
@@ -1381,7 +1381,45 @@ mfxStatus CoreDoSWFastCopy(mfxFrameSurface1 *pDst, mfxFrameSurface1 *pSrc, int c
...
+    case MFX_FOURCC_Y210:
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
```
Comment (dvrogozh) [inline _studio/shared/src/libmfx_core.cpp]: And the whole block presents already below, line ~1470: https://github.com/Intel-Media-SDK/MediaSDK/pull/1788/commits/4ac1843e24f90d84a1e25559efafa3bf42784f84#diff-fae1222e835b7a0eea6af66261c4d4d0R1470

File: _studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp (hunk 1 above)
Comment (saosipov) [reply _studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp]: For 12b encoding both Luma and Chroma TargetBitdepth should be 12 (at least other behavior is not confirmed yet) while for 10b encoding it is allowed to have LumaTargetBitdepth=10 and ChromaTargetBitdepth=8
Commented lines can be easily removed.

**Pull request merged**